db/migrate/20150930183738_migrate_content_hosts.rb

```diff
@@ -1,14 +1,19 @@
 class MigrateContentHosts < ActiveRecord::Migration
-  class Katello::HostCollectionAssociation < ActiveRecord::Base
+  class Katello::SystemRepository < ActiveRecord::Base
```

Comment:
This is going to modify the actual Katello::SystemRepository class. It is better to just remove "Katello::" SystemRepository and set the table name similar to the template in http://projects.theforeman.org/issues/13266